src/component/page/bruker-detaljer/deltaker-detaljer/endre-deltaker-modal/AarsakSelector.tsx

```diff
@@ -33,8 +36,24 @@ export const AarsakSelector = ({ tittel, onAarsakSelected }: AarsakSelectorProps
 			<AarsakRadio aarsakType={DeltakerStatusAarsakType.TRENGER_ANNEN_STOTTE} />
 			<AarsakRadio aarsakType={DeltakerStatusAarsakType.IKKE_MOTT} />
 			<AarsakRadio aarsakType={DeltakerStatusAarsakType.UTDANNING} />
+			{ skalViseOppfyllerIkkeKrav && <AarsakRadio aarsakType={DeltakerStatusAarsakType.OPPFYLLER_IKKE_KRAVENE} >
+				{visBeskrivelseOppfyllerIkkeKravene ? <>
+					<TextField
+						onChange={e => settBeskrivelse(e.target.value)}
+						value={beskrivelse ?? ''}
```

Comment:
Det er kanskje med vilje at beskrivelsen bruker samme state i de to boksene..

Reply:
har ikke tenkt veldig bevisst på det nei 😅 Husk at jeg kan ikke frontend i det hele tatt. Men man kan vel ikke ha flere beskrivelser uansett?

Reply:
Har fått masse hjelp av Tormod her i forbindelse med å trekke dette ut i en egen komponent, og da har jeg også fått hjelp til å nulle ut staten når du går fra det ene valget til det andre (for du hadde rett i at det ble et problem 😅 )